src/inverseRobot.js

```diff
@@ -8,6 +8,18 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inverted = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (inverted[value] !== undefined) {
+      return null;
+    }
+    inverted[value] = key;
```

Comment:
[CODE STYLE] - use correct check for property presence in object. Some properties can be present, but still contain falsy value.
![image](https://github.com/mate-academy/js_inverse_robot/assets/63243136/d951acd4-86bb-40b1-af6f-e0bd5487c3a5)